internal/services/oracle/oracle_autonomous_database_backup_data_source_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package oracle_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+)
+
+type AutonomousDatabaseBackupDataSourceTest struct{}
+
+func TestAccAutonomousDatabaseBackupDataSource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_oracle_autonomous_database_backup", "test")
+	r := AutonomousDatabaseBackupDataSourceTest{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
```

Comment:
Tests is failing due to limited region support, can we add the below to the test?

```suggestion
	// TODO - Remove this override when resource is rolled out to westeurope - currently only supported in limited regions
	data.Locations.Primary = "eastus2"
	
	data.DataSourceTest(t, []acceptance.TestStep{
```

Reply:
@wyattfry thank you for your attention to this PR , I will address you comments. However I don't understand why we need to add zone override when hashicorp policy for oracle doesn't require to trigger accTest as @WodansSon mentioned. Thank you!